### ISSUE TITLE
AUTOGENERATED_home_THIS_SHOULDN'T_SHOW_UP_ANYWHERE shows up immediately

### ISSUE THREAD

**Issue by 1j01** [witheve/Eve#351]
* Installed eve (`eve@0.0.0`? `master`?) on Windows 10
* Opened `http://localhost:3000`
* Got `http://localhost:3000/#/home/AUTOGENERATED_home_THIS_SHOULDN'T_SHOW_UP_ANYWHERE` in the URL

**Comment by timsandgren** [comment]
Yup.

**Comment by cmontella** [comment]
This behavior is fine. "This shouldn't show up anywhere" was appended to built-in entities to make them recognizable. It should be changed maybe, but they're not part of the system anymore.

**Comment by 1j01** [comment]
To make them recognizable as a bug? Maybe you should change it to `SOMETHING_WENT_WRONG_AAAAAAAAAAAAGH_NO_EVERYTHING_IS_FINE_DON'T_WORRY_ABOUT_IT`

**Comment by joshuafcole** [comment]
For some clarity on what's going on here -- the provided test datasets that we ship with use deterministically generated ids to make them easy to map for the testing suite. Originally, we showed ids as the names of entities as a time saver, but once actual UUID v4's entered the mix (for user generated content) that was obviously unacceptable. When that happened, I needed to move through the whole system and update everything that referenced entities to either use the id (for lookup) or the name (for display).

To make sure I didn't miss anything, I changed the deterministically assigned UUID pattern to be very *ahem* recognizable. That those ids then showed up in production urls is a known bug (those deterministic ids were supposed to be replaced with regular UUIDs) but the priority was low and we'd already determined taken our lessons from this demo into account as we started working on the next one. Changing this to fix the symptoms is a one liner (which changes the id generation function to map to actual UUIDs) but this breaks the smoke testing paths which would need to be updated to use a dynamic mapping to entities. We decided in the end that leaving a cosmetic bug in the prototype was a better state than leaving it untestable.
